corehq/apps/programs/views.py

```diff
@@ -71,10 +73,11 @@ def get(self, request, *args, **kwargs):
         }), 'text/json')
 
 
+@method_decorator(use_bootstrap5, name='dispatch')
```

Comment:
> Can build_requirejs.py detect this...

I think it's a stretch to make `build_requirejs` connect HTML templates to the python view that renders them. Maybe not impossible, but the benefit seems pretty small, assuming I automate a B5 indicator on the `requirejs_main` tag. We're already in a situation where we can have conflicting indicators: a page can use this decorator, but its template can still extend a B3 template.

> Alternately, could a B5 indicator on the requirejs_main tag make this decorator obsolete?

This isn't workable because there will be B5 pages that don't use requirejs_main, since not all of HQ is on requirejs.